Dockerfile

```diff
@@ -75,9 +78,14 @@ FROM centos:7 AS centos7-build-base
 RUN set -x; yum -y update && \
 		yum -y install git systemd \
 		autoconf libtool libcurl-devel libtool-ltdl-devel openssl-devel yajl-devel \
-		gcc gcc-c++ make bison flex file systemd-devel zlib-devel gtest-devel rpm-build java-11-openjdk-devel \
+		gcc gcc-c++ make bison flex file systemd-devel zlib-devel gtest-devel rpm-build \
 		expect rpm-sign zip
-		ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk/
+		COPY --from=openjdk-install /tmp/OpenJDK17U.tar.gz /tmp/OpenJDK17U.tar.gz
+		RUN set -xe; \
+			mkdir -p /usr/local/java-17-openjdk && \
+			tar -xf /tmp/OpenJDK17U.tar.gz -C /usr/local/java-17-openjdk --strip-components=1
```

Comment:
[Optional] We run this command verbatim whenever we use the `openjdk-install`. container. Why not factor it out into that container, and then copy `/usr/local/java-${OPENJDK_MAJOR_VERSION}-openjdk` from it directly?